File: bean-validation-rest/src/main/java/com/github/yukihane/java/beanvalidationrest/MyConfig.java
package com.github.yukihane.java.beanvalidationrest;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.context.MessageSource;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;

    @Configuration
    public class MyConfig {

        @Autowired
        private MessageSource messageSource;

        @Bean
    public LocalValidatorFactoryBean localValidatorFactoryBean() {
            final LocalValidatorFactoryBean factoryBean = new LocalValidatorFactoryBean();
            factoryBean.setValidationMessageSource(messageSource);
            return factoryBean;
        }
    }

File: spring/hello-thymeleaf-layout-dialect/src/main/kotlin/com/github/yukihane/examples/hello_thymeleaf_layout_dialect/MyConfig.kt
package com.github.yukihane.examples.hello_thymeleaf_layout_dialect

import nz.net.ultraq.thymeleaf.layoutdialect.LayoutDialect
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration


@Configuration(proxyBeanMethods = false)
class MyConfig {

    @Bean
    fun layoutDialect(): LayoutDialect {
        return LayoutDialect()
    }
}
